ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Byron.hs

```diff
@@ -300,6 +300,16 @@ instance (ByronGiven, Typeable cfg, ConfigContainsGenesis cfg, NoUnexpectedThunk
         where
           slot = convertSlot (CC.Block.cvsLastSlot state)
 
+deriving instance ByronGiven => NoUnexpectedThunks (LedgerState (ByronBlock cfg))
```

Comment:
Do you want `instance` rather than `deriving instance` here, if the intention is to use the generic instance? In this case I think it will do the same thing, but in general the pattern elsewhere is to use an empty instance directly where we want to use generics

Reply:
You're right, will fix.